### PR TITLE
Homebrew sbt for MacOS

### PR DESCRIPTION
**Pull request by schoeberl**
### Contributor Checklist

- [ ] Did you add Scaladoc to every public function/method?
- [ ] Did you add at least one test demonstrating the PR?
- [ ] Did you delete any extraneous printlns/debugging code?
- [ ] Did you specify the type of improvement?
- [ ] Did you add appropriate documentation in `docs/src`?
- [ ] Did you request a desired merge strategy?
- [ ] Did you add text to be included in the Release Notes for this change?

#### Type of Improvement

- Documentation or website-related

#### Desired Merge Strategy

- Squash: The PR will be squashed and merged (choose this if you have no preference).
- Rebase: You will rebase the PR onto master and it will be merged with a merge commit.

### Reviewer Checklist (only modified by reviewer)
- [ ] Did you add the appropriate labels? (Select the most appropriate one based on the "Type of Improvement")
- [ ] Did you mark the proper milestone (Bug fix: `3.5.x`, `3.6.x`, or `5.x` depending on impact, API modification or big change: `6.0`)?
- [ ] Did you review?
- [ ] Did you check whether all relevant Contributor checkboxes have been checked?
- [ ] Did you do one of the following when ready to merge:
  - [ ] Squash: You/ the contributor `Enable auto-merge (squash)`, clean up the commit message, and label with `Please Merge`.
  - [ ] Merge: Ensure that contributor has cleaned up their commit history, then merge with `Create a merge commit`.

### CODE REVIEW

File: docs/src/installation.md [hunk 1]
```diff
@@ -157,6 +157,11 @@ Using [MacPorts](https://www.macports.org):
 ```sh
 sudo port install sbt
 ```
+or with [Homebrew](https://brew.sh/)
```
Comment (jackkoenig) [inline docs/src/installation.md:160]:
```suggestion

Using [Homebrew](https://brew.sh/):
```

Stylistic nit.

**Review by jackkoenig — COMMENTED** [review]
Thank you for the grammatical improvements.

Have you tested sbt with homebrew lately? I ask because I originally didn't add Homebrew intentionally due to the warning about it on the SBT docs: https://www.scala-sbt.org/download#homebrew

I'm not against adding it, I just want to make sure that problem doesn't apply, we need to make sure SBT isn't overriding the users's or they may have surprising behavior.

File: docs/src/installation.md (hunk 1 above)
Comment (schoeberl) [reply docs/src/installation.md:160]: I am using the Homebrew version of sbt. Just installed it and reinstalled it to check. The issue is that brew then installs Java 21, which is broken with sbt (similar to the issue you link to). After uninstalling Java 21 from brew (`brew uninstall --ignore-dependencies java`) it works again. So I don't know what is best. As many MacOs users are using brew, I would include it. Maybe add the warning about Java 21.

File: docs/src/installation.md [hunk 2]
```diff
@@ -157,6 +157,17 @@ Using [MacPorts](https://www.macports.org):
...
+
+```sh
+brew install sbt
+```
+
+Note that `brew` installs the latest version of Java (21) as a dependency of `sbt`. However, that version is broken. To revert to an older Java version installed (not with `brew`), remove the `brew` version of Java with:
+
+```sh
+uninstall --ignore-dependencies java
+```
```
Comment (jackkoenig) [inline docs/src/installation.md:170]:
````suggestion

::: warning

Note that `brew` installs the latest version of Java as a dependency of `sbt` even though that version of Java is not actually supported by SBT.
Users are advised to remove the `brew` version of Java with the following command:

```sh
brew uninstall --ignore-dependencies java
```

:::
````
I'm gonna up the urgency of comment a little bit to hopefully save some users some grief.

**Review by jackkoenig — APPROVED** [review]

**Pull request merged**